1754508743239	Sync.LogManager	DEBUG	Flushing file log
1754508743242	Sync.LogManager	DEBUG	Log cleanup threshold time: 1753644743242
1754508743286	Sync.LogManager	DEBUG	Done deleting files.
1754508826430	Sync.SyncScheduler	DEBUG	Network link status changed to "down". Offline?: false
1754508826431	Sync.SyncScheduler	DEBUG	Clearing sync triggers and the global score.
1754508826432	Sync.SyncScheduler	DEBUG	Clearing sync triggers and the global score.
1754508826435	Sync.SyncScheduler	DEBUG	Network link status changed to "changed". Offline?: false
1754508826966	Sync.SyncScheduler	DEBUG	Network link status changed to "changed". Offline?: false
1754508829277	Sync.SyncScheduler	DEBUG	Network link status changed to "up". Offline?: false
1754508829277	Sync.SyncScheduler	DEBUG	Network link looks up. Syncing.
1754508829279	Sync.SyncScheduler	DEBUG	Clearing sync triggers and the global score.
1754508829280	Sync.SyncScheduler	DEBUG	Network link status changed to "changed". Offline?: false
1754508829281	Sync.Service	DEBUG	User-Agent: Zen/1.14.9b (Linux x86_64) FxSync/1.143.0.20250727064442.desktop
1754508829281	Sync.Service	INFO	Starting sync at 2025-08-06 22:33:49 in browser session s2EO8P2iTOUf
1754508829281	Sync.Service	DEBUG	In sync: should login.
1754508829281	Sync.Service	INFO	User logged in successfully - verifying login.
1754508829281	Sync.SyncAuthManager	DEBUG	unlockAndVerifyAuthState has no FxA user
1754508829282	Sync.Status	DEBUG	Status.login: error.login.reason.no_username => error.login.reason.no_username
1754508829282	Sync.Status	DEBUG	Status.service: service.client_not_configured => service.client_not_configured
1754508829282	Sync.Service	DEBUG	Fetching unlocked auth state returned error.login.reason.no_username
1754508829282	Sync.ErrorHandler	ERROR	Sync encountered a login error
1754508829282	Sync.SyncScheduler	DEBUG	Clearing sync triggers and the global score.
1754508829282	Sync.SyncScheduler	DEBUG	Clearing sync triggers and the global score.
1754508829282	Sync.Service	DEBUG	Exception calling WrappedLock: Error: Login failed: error.login.reason.no_username(resource://services-sync/service.sys.mjs:1051:15) JS Stack trace: onNotify@service.sys.mjs:1051:15
1754508829282	Sync.Service	DEBUG	Not syncing: login returned false.
